packages/sdk/src/lend/providers/morpho/index.ts

```diff
@@ -19,27 +18,22 @@ import { LendProvider } from '../../provider.js'
 import { findBestVaultForAsset, getVault, getVaults } from './sdk.js'
 
 /**
- * Supported networks for Morpho lending
+ * Supported network IDs for Morpho lending
+ * @description Array of chain IDs where Morpho is available
  */
-export const SUPPORTED_NETWORKS = {
-  UNICHAIN: {
-    chainId: 130,
-    name: 'Unichain',
-    morphoAddress: '0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb' as Address,
-  },
-  BASE_SEPOLIA: {
-    chainId: baseSepolia.id,
-    name: 'Base Sepolia',
-    morphoAddress: '0xBBBBBbbBBb9cC5e90e3b3Af64bdAF62C37EEFFCb',
-  },
-} as const
+export const SUPPORTED_NETWORK_IDS = [
+  mainnet.id, // 1
+  unichain.id, // 130
+  base.id, // 8453
+  baseSepolia.id, // 84532
+] as const
```

Comment:
this should be typed as `SUPPORTED_NETWORK_IDS`: `SupportedChainIds[]` that way it will not be possible to include a chain id that is not supported by the sdk

Reply:
Morpho's sdk actually exports a `ChainId` enum with all of the networks the have markets on, so i just updated this to use their type.